Lib/fontTools/feaLib/builder.py

```diff
@@ -1259,18 +1263,21 @@ def build(self):
             self.setLookAheadCoverage_(suffix, st)
             self.setInputCoverage_(glyphs, st)
 
-            st.PosCount = len([l for l in lookups if l is not None])
+            st.PosCount = 0
             st.PosLookupRecord = []
-            for sequenceIndex, l in enumerate(lookups):
-                if l is not None:
-                    if l.lookup_index is None:
-                        raise FeatureLibError('Missing index of the specified '
-                            'lookup, might be a substitution lookup',
-                            self.location)
-                    rec = otTables.PosLookupRecord()
-                    rec.SequenceIndex = sequenceIndex
-                    rec.LookupListIndex = l.lookup_index
-                    st.PosLookupRecord.append(rec)
+            for sequenceIndex, ll in enumerate(lookups):
+                if ll is not None:
+                    if type(ll) is not list: ll = [ll]
```

Comment:
`isinstance()`?

Reply:
May  be also a new line after the colon and better variable names than `l` and `ll` (they remind me of my peronal code, `a`, `aa`, `aaa`, `aaaa`).